### PR TITLE
cmd: add --log-level CLI flag

### PR DESCRIPTION
**Pull request by lgfa29**
Allow specifying the log level directly as the global CLI flag `--log-level`.

~Draft for now since it requires https://github.com/loopholelabs/logging/pull/7 and https://github.com/loopholelabs/logging/pull/8.~ Merged!

### CODE REVIEW

File: pkg/command/testing.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	cfgFile string
+	logFile string
+
+	// Common configuration values.
+	Format  string
+	Debug   bool
+	NoColor bool `mapstructure:"no-color"`
+
+	// Custom configuration values.
+	MyConfig string `mapstructure:"my-config"`
```
Comment (lgfa29) [inline pkg/command/testing.go:93]: I was using these in a WIP set of config tests that I may push in the future, but they're not exactly necessary in this PR. I can remove them for now, but they also don't really affect anything.

Comment (ShivanshVij) [reply pkg/command/testing.go:93]: Format,Debug, NoColor are all fine, but MyConfig seems out of place and might confuse, so let's get rid of that for now.

**Review by ShivanshVij — APPROVED** [review]
LGTM!

File: pkg/command/testing.go (hunk 1 above)
Comment (lgfa29) [reply pkg/command/testing.go]: Yeah, sounds good. Removed in https://github.com/loopholelabs/cmdutils/pull/15/commits/570b233e5d7217ac42e30385be4eeed8c07e5993

**Pull request merged**